package.json

```diff
@@ -24,13 +24,17 @@
   "scripts": {
     "start": "ember serve",
     "build": "ember build",
-    "lint:js": "eslint .",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:js": "eslint --ext ts,js .",
+    "lint:types": "tsc --noEmit",
     "lint-staged": "lint-staged",
     "tests-node": "mocha \"tests-node/**/*.js\"",
     "test": "yarn run tests-node && ember test",
     "test-node": "yarn tests-node",
     "testall": "yarn run tests-node && ember try:each",
-    "prettier": "prettier --single-quote --print-width 120 --write \"{blueprints,config,lib,app,addon,addon-test-support,tests,tests-node}/**/*.js\""
+    "prettier": "prettier --single-quote --print-width 120 --write \"{blueprints,config,lib,app,addon,addon-test-support,tests,tests-node}/**/*.js\"",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean"
```

Comment:
@jasonmit this is where the magic happens :)

prepack / postpack work more reliably than pre/post-publish and will be used by ember-cli-typescript@4 by default.

Reply:
Weird - I didn't see this in the github diff, sorry!